### DISCUSSION TITLE
How do I tell the machine is farming or not

### DISCUSSION THREAD

**Discussion by aural227** [Chia-Network/chia-blockchain#4514] [Support]
Hello,

I build three windows 10 machines last week, Machine A, B, and C; and they are in the same subnet.
The latest Chia GUI (1.1.15) has been installed in all threes, and the same key is used. 
By following the installation guide, ports has been forwarded to Machine A, since it is planned to be a farmer and harvester machine. UPNP is also disabled on Machine B and C, since these two are planned to be a harvester machine only, and all plots created will be stored at the shared drive on Machine A. 
The question I have at this moment: can anyone tell me how I know there is only Machine A is farming and the other two are not? I saw weird things, which makes wonder if the machines are set up properly. First, whenever I go to "Farm" Tab at each machine, the status on "plots passed filter" are changing at all machines. Second, if I go to "Plots" Tab, each machine shows the status is farming, even after I exit chia at Machine A, the status on other two is still showing "Farming".

Thank you and I appreciate all suggestions.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
